tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -195,6 +201,9 @@ var _ = ginkgo.Describe("VM backup and restore tests", ginkgo.Ordered, func() {
 		if v != nil && wasInstalledFromTest {
 			v.EnsureVirtRemoval()
 		}
+
+		err := v.Clientset.StorageV1().StorageClasses().Delete(context.TODO(), "test-gp3-csi-immediate", metav1.DeleteOptions{})
```

Comment:
can we use `UninstallApplication` function here instead?

Reply:
Yes, so far this seems to be working fine for me locally.